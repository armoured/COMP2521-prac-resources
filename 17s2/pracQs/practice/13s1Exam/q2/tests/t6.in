22 55 88 11 66 77 33 99 44
