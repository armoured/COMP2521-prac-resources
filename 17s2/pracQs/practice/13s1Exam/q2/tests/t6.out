Original List: [22, 55, 88, 11, 66, 77, 33, 99, 44]
Reversed List: [44, 99, 33, 77, 66, 11, 88, 55, 22]
Length before: 9, Length after: 9
